src/Handler/CurlFactory.php

```diff
@@ -453,34 +458,34 @@ private function applyHandlerOptions(EasyHandle $easy, array &$conf): void
         if (isset($options['cert'])) {
             $cert = $options['cert'];
             if (\is_array($cert)) {
-                $conf[CURLOPT_SSLCERTPASSWD] = $cert[1];
+                $conf[\CURLOPT_SSLCERTPASSWD] = $cert[1];
                 $cert = $cert[0];
             }
             if (!\file_exists($cert)) {
                 throw new \InvalidArgumentException(
                     "SSL certificate not found: {$cert}"
                 );
             }
-            $conf[CURLOPT_SSLCERT] = $cert;
+            $conf[\CURLOPT_SSLCERT] = $cert;
         }
 
         if (isset($options['ssl_key'])) {
             if (\is_array($options['ssl_key'])) {
                 if (\count($options['ssl_key']) === 2) {
-                    list($sslKey, $conf[CURLOPT_SSLKEYPASSWD]) = $options['ssl_key'];
+                    [$sslKey, $conf[\CURLOPT_SSLKEYPASSWD]] = $options['ssl_key'];
                 } else {
-                    list($sslKey) = $options['ssl_key'];
+                    [$sslKey] = $options['ssl_key'];
                 }
             }
 
-            $sslKey = isset($sslKey) ? $sslKey: $options['ssl_key'];
+            $sslKey = $sslKey?? $options['ssl_key'];
```

Comment:
curious, why didn't the fixer ensure a space between `$sslKey` and `??` here?

Reply:
oh, the `binary_operator_spaces` rule is not enabled by the `@PSR2` ruleset...

Reply:
Do you mind submitting a followup PR?

Reply:
@gmponos: done: #2688